bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java

```diff
@@ -75,27 +75,27 @@ public DanfossAirUnitHandler(final Thing thing, final TimeZoneProvider timeZoneP
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        Channel channel = Channel.getByName(channelUID.getIdWithoutGroup());
+
         if (command instanceof RefreshType) {
-            updateAllChannels();
-        } else {
-            try {
-                DanfossAirUnit localAirUnit = this.airUnit;
-                if (localAirUnit != null) {
-                    Channel channel = Channel.getByName(channelUID.getIdWithoutGroup());
-                    DanfossAirUnitWriteAccessor writeAccessor = channel.getWriteAccessor();
-                    if (writeAccessor != null) {
-                        updateState(channelUID, writeAccessor.access(localAirUnit, command));
-                    }
-                } else {
-                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.NONE,
-                            "@text/offline.connection-not-initialized");
-                    return;
+            updateChannel(channel, true);
+            return;
+        }
+
+        try {
+            DanfossAirUnit airUnit = this.airUnit;
+            if (airUnit != null) {
+                DanfossAirUnitWriteAccessor writeAccessor = channel.getWriteAccessor();
+                if (writeAccessor != null) {
+                    updateState(channelUID, writeAccessor.access(airUnit, command));
                 }
-            } catch (IllegalArgumentException e) {
-                logger.debug("Ignoring unknown channel id: {}", channelUID.getIdWithoutGroup(), e);
-            } catch (IOException ioe) {
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR, ioe.getMessage());
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.NONE,
+                        "@text/offline.connection-not-initialized");
+                return;
```

Comment:
I'd prefer to have this removed. Apart from being superfluous in this specific case, its easily missed if someone else is adding more code below later. I'm not a friend of early returns in cases other than the one above in L80 where it is really obvious. The deeper down in nested structures you have a return the more likely it is to be missed.

Reply:
I agree, and have removed it. It was not added by this PR. 😉